core/src/databases/table_schema.rs

```diff
@@ -292,31 +287,31 @@ impl TableSchema {
         field_names: &Vec<&String>,
         row: &Row,
     ) -> Result<Vec<SqlParam>> {
-        match row.content().as_object() {
-            None => Err(anyhow!("Row content is not an object")),
-            Some(object) => field_names
-                .iter()
-                .map(|col| match object.get(*col) {
-                    Some(Value::Bool(b)) => Ok(SqlParam::Bool(*b)),
-                    Some(Value::Number(x)) => {
-                        if x.is_i64() {
-                            Ok(SqlParam::Int(x.as_i64().unwrap()))
-                        } else if x.is_f64() {
-                            Ok(SqlParam::Float(x.as_f64().unwrap()))
-                        } else {
-                            Err(anyhow!("Number is not an i64 or f64"))
-                        }
+        field_names
+            .iter()
+            .map(|col| match row.value().get(*col) {
+                Some(Value::Bool(b)) => Ok(SqlParam::Bool(*b)),
+                Some(Value::Number(x)) => {
+                    if x.is_i64() {
+                        Ok(SqlParam::Int(x.as_i64().unwrap()))
```

Comment:
we should also fix those unwraps (i know it's not your code and we just checked them but still not a great practice as it's a good way to shoot ourselves in the foot with unscrutable panics)